app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/received/ReceivedScreen.kt

```diff
@@ -40,34 +42,21 @@ fun ReceivedScreen(
     getSelectedTransferUuid: () -> String?,
     transfersViewModel: TransfersViewModel = hiltViewModel<TransfersViewModel>(),
 ) {
-    val transfers by transfersViewModel.receivedTransfers.collectAsStateWithLifecycle()
-    val areTransfersEmpty = false // TODO: Use the next line instead when available :
-    // val areTransfersEmpty by remember { derivedStateOf { transfers?.isEmpty() == true } }
 
-    ReceivedScreen(
-        navigateToDetails = navigateToDetails,
-        getSelectedTransferUuid = getSelectedTransferUuid,
-        areTransfersEmpty = { areTransfersEmpty },
-    )
-}
+    val transfers by transfersViewModel.receivedTransfers.collectAsStateWithLifecycle()
+    val areTransfersEmpty by remember { derivedStateOf { transfers.isEmpty() } }
 
-@Composable
-private fun ReceivedScreen(
```

Comment:
You shouldn't remove this component as it breaks the preview because of the viewModel usage